src/tools/perf/perftest_run.c

```diff
@@ -25,6 +25,7 @@ void print_progress(char **test_names, unsigned num_names,
                     unsigned flags, int final, int is_server,
                     int is_multi_thread)
 {
+    UCS_STRING_BUFFER_ONSTACK(strb, 256);
```

Comment:
will 256 be always sufficient? Maybe 1024?

Reply:
it's supposed to be one line of output, shouldn't be that long